### PR TITLE
Added dropdown on avatar image and avatar buttons in rhinostyle

### PR DESCRIPTION
**Pull request by rosh-ttn**
To achieve dropdown functionality on images e.g profile settings dropdown on profile image, it was important to pass avatar as props in dropdowns component.
Also, to maintain button styles, added avatar buttons in rhinostyle/buttons.
Jira Link: https://rhinogram.atlassian.net/browse/RHIN-839

✔︎ I've tested in all browsers (IE11, Firefox, Safari, Chrome, and iOS)

### What should this PR do?
Add avatar image to buttons or dropdowns.

### What is the acceptance criteria?
Not Applicable
[If there are UI changes, include a before and after screenshot.]

### CODE REVIEW

File: src/scripts/components/Dropdown.jsx [hunk 1]
```diff
@@ -216,7 +231,7 @@ Dropdown.propTypes = {
   disableScroll: PropTypes.bool,
   hideCaret: PropTypes.bool,
   hideActive: PropTypes.bool,
-  icon: PropTypes.string,
+  icon: customValidator,
```
Comment (mahimaag) [inline src/scripts/components/Dropdown.jsx]: Why we are using customValidator while defining propTypes?

File: dist/scripts/rhinostyle.min.js [hunk 2]
```diff
@@ -9372,27 +9372,27 @@ var locationsAreEqual = exports.locationsAreEqual = function locationsAreEqual(a
...
+	// This works if the window reference is available
+	if(typeof window === "object")
+		g = window;
+}
+
+// g can still be undefined, but nothing to do about it...
+// We return undefined, instead of nothing here, so it's
+// easier to handle this case. if(!global) { ...}
+
+module.exports = g;
```
Comment (mahimaag) [inline dist/scripts/rhinostyle.min.js]: Why we are changing these line of codes?

File: dist/scripts/rhinostyle.min.js [hunk 3]
```diff
@@ -41540,7 +41540,8 @@ var Button = function (_React$Component) {
           block = _props.block,
           className = _props.className,
           disabled = _props.disabled,
-          iconOnly = _props.iconOnly,
+		  iconOnly = _props.iconOnly,
+		  avatarOnly = _props.avatarOnly,
```
Comment (mahimaag) [inline dist/scripts/rhinostyle.min.js]: Indentation formatting.

**Comment by rosh-ttn** [comment]
reopened the issue in a different PR.

**Pull request closed without merging**